requirements.txt

```diff
@@ -6,6 +6,7 @@ six
 # methods
 scipy
 periodictable
+h5py
```

Comment:
I know that the `requirements.txt` / `setup.py` / Travis (optional) dependencies are messy right now, but can you remove h5py from here for now?

Also, is it automatically installed as a Horton dependency on Travis?

Reply:
In Python 2.7 machine, `h5py` is installed as a horton dependency. In Python 3.x machines, though, it isn't since horton is not installed there. Should h5py be moved to conda configuration then?

Reply:
For now, yes.